src/TimeoutMigrationTool/ASQ/DelayedDeliveryTableNameProvider.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace Particular.TimeoutMigrationTool.ASQ
+{
+    using System;
+    using System.Security.Cryptography;
+    using System.Text;
+
+    public class DelayedDeliveryTableNameProvider : IProvideDelayedDeliveryTableName
```

Comment:
The interface seems overkill to me. It doesn't look like we really need this extensibility

Reply:
It's only here so that it can be tested :/